dapps/tests/teller-contracts/.gitignore

```diff
@@ -0,0 +1,8 @@
+.embark
```

Comment:
Because of the following two higher-up files, this `.gitignore` is not needed and can be deleted:

https://github.com/embarklabs/embark/blob/feat/teller-contracts-2/dapps/.gitignore

https://github.com/embarklabs/embark/blob/feat/teller-contracts-2/.gitignore